homeassistant/components/bluetooth_tracker/device_tracker.py

```diff
@@ -173,10 +173,10 @@ async def perform_bluetooth_update() -> None:
                     rssi = await hass.async_add_executor_job(client.request_rssi)
                     client.close()
 
-                tasks.append(see_device(hass, async_see, mac, friendly_name, rssi))
+                tasks.append(asyncio.create_task(see_device(hass, async_see, mac, friendly_name, rssi)))
 
             if tasks:
-                await asyncio.wait(tasks)
+                await asyncio.wait(tasks, return_when=asyncio.ALL_COMPLETED)
```

Comment:
```suggestion
                await asyncio.wait(tasks)
```

This change isn't needed. ALL_COMPLETED is the default behavior

https://docs.python.org/3/library/asyncio-task.html#asyncio.wait